### PR TITLE
Create Acceptable Use Policy

### PR DESCRIPTION
**Pull request by EThom73**
This is a rewrite and relocation of the old acceptable use policy. This has had the language updated and made more "Made Tech".

### CODE REVIEW

File: guides/security/acceptable_use_policy.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+I will use the Made Tech service for the purpose it is intended for at all times, and not do anything knowingly that could harm the integrity or reputation of Made Tech.
+
+## I will use any Made Tech issued devices properly
+I understand Made Tech issued devices are issued for work purposes and I agree to return them for inspection as and when required. I will return it on leaving the employment of Made Tech or when it is no longer required unless otherwise agreed with Made Tech prior to leaving.
+
+## Personal Use
+I will exercise good judgement at all times regarding the reasonableness of any personal use
+
+## I will enable  2-factor authentication where this is required
```
Comment (chrisblackburn) [inline guides/security/acceptable_use_policy.md]: required or available?

File: guides/security/acceptable_use_policy.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+2-factor authentication (2FA) requires two forms of authentication, often a password and a code, sometimes sent via SMS or an application. It helps protect user accounts in the case that your password is somehow exposed. 
+
+## I will ensure my laptop isn’t left unattended or unlocked
+Lock your screen or put your computer to sleep when you’re not around. Colleagues might not have the same permissions as you and visitors might be wandering around where you’re working. When not in use ‘at work’, you can store in the laptop lockers provided or ‘at home’, in a safe place where it can’t be seen. If you are using your device in a public place, don’t leave it unattended and be wary of those who may be watching over your shoulder. 
+
+## I will ensure I report the arrival of guests and visitors promptly
```
Comment (chrisblackburn) [inline guides/security/acceptable_use_policy.md]: I think this is covered elsewhere? Also feels strange as part of an AUP, which in my mind is more about how I use the devices/services provided for my work

File: guides/security/acceptable_use_policy.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+## If my device is stolen or missing, I immediately report it
+Lots of information can be extracted from stolen or misplaced laptops, even in some cases when the disk has been encrypted. Report a missing machine quickly, better safe than sorry. We use remote device management to remotely wipe data from your device in the event that it is stolen. You should report all stolen items to the IMS Manager or CSO, who will record the incident. 
+
+## I will backup my work in the right place
+Information saved on a device is at risk of loss or theft and will not be backed up. To protect against theft, laptops are encrypted. Made Tech provides back up facilities within Google (Drive). Each team at Made Tech has their own Team Drive and it is the responsibility of those teams to maintain what is kept in those drives. All data must be copied to these locations to ensure there is no data loss in the event of your laptop becoming unusable.
```
Comment (chrisblackburn) [inline guides/security/acceptable_use_policy.md]: `Google (Drive)` is different to how we refer to G-Suite elsewhere
We could say `Made Tech backs up data stored on G-Suite, GitHub, and Slack.`

File: guides/security/acceptable_use_policy.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+Keep passwords secure and do not share accounts with anyone else. You are responsible for the security of your passwords and account.
+
+## I will not share my laptop with anyone else, including under a separate user account
+The laptop you use for working should not be used by anyone else; for example a partner having a separate user account or sharing a generic login.
+
+## I will not open unfamiliar email attachments, and I will always check the actual address for links in emails, even from trusted colleagues
+Phishing and in particular spear phishing (which targets specific individuals within a specific organisation) is a genuine threat with recorded attacks everyday.
+
+## I report anything that could be a security incident
+ Better safe than sorry. If you experience or see anything suspicious,or anything that you know breaks one of our security policies, report it to either the IMS manager or CSO. As a general rule, if you are not sure whether to report something, report it anyway. We'd always rather know.
```
Comment (chrisblackburn) [inline guides/security/acceptable_use_policy.md]: The second `better safe than sorry` in this doc. A needed Colinism?

Comment (chrisblackburn) [reply guides/security/acceptable_use_policy.md]: Also missing a space after a comma

File: guides/security/acceptable_use_policy.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+## I am careful when using bluetooth connections
+Bluetooth is enabled on your device and can be used safely within an office or your home. Its use outside of those environments is not recommended as they may be easily compromised.
+
+## I use my email account responsibly
+Use of your company email addresses should be for work-related content only. Do not use your email address to login to non-work related websites or social media
+
+## I am conscious of my web footprint
+Be aware that your online profiles and activity may link back to Made Tech so be aware of this when you publish anything online
+
+## I am responsible when personal devices on Made Tech networks
```
Comment (chrisblackburn) [inline guides/security/acceptable_use_policy.md]: I think this is missing a word?

File: guides/security/acceptable_use_policy.md (hunk 1 above)
Comment (EThom73) [reply guides/security/acceptable_use_policy.md]: amended

File: guides/security/acceptable_use_policy.md (hunk 2 above)
Comment (EThom73) [reply guides/security/acceptable_use_policy.md]: removed

File: guides/security/acceptable_use_policy.md (hunk 3 above)
Comment (EThom73) [reply guides/security/acceptable_use_policy.md]: amended

File: guides/security/acceptable_use_policy.md (hunk 4 above)
Comment (EThom73) [reply guides/security/acceptable_use_policy.md]: Actually it's a me-ism. Makes me sound like my mother. Removed.

File: guides/security/acceptable_use_policy.md (hunk 5 above)
Comment (EThom73) [reply guides/security/acceptable_use_policy.md]: added "using"

File: guides/security/acceptable_use_policy.md [hunk 6]
```diff
@@ -45,7 +42,7 @@ The laptop you use for working should not be used by anyone else; for example a
 Phishing and in particular spear phishing (which targets specific individuals within a specific organisation) is a genuine threat with recorded attacks everyday.
 
 ## I report anything that could be a security incident
- Better safe than sorry. If you experience or see anything suspicious,or anything that you know breaks one of our security policies, report it to either the IMS manager or CSO. As a general rule, if you are not sure whether to report something, report it anyway. We'd always rather know. 
+If you experience or see anything suspicious,or anything that you know breaks one of our security policies, report it to either the IMS manager or CSO. As a general rule, if you are not sure whether to report something, report it anyway. We'd always rather know.
```
Comment (chrisblackburn) [inline guides/security/acceptable_use_policy.md]: Just one missing space after the comma and I think this one is good to go

**Pull request merged**